README.md

```diff
@@ -251,6 +253,178 @@ To run other tests:
 ./scripts/run_tests.sh
 ```
 
+## Deployment
+
+Instructions on how to set up the pipeline on a production environment.
+
+### Dependencies
+
+- Python >= 3.6,
+- Package `python3.6-venv` (or higher version, depending on the version of Python) installed, 
+- Git,
+- An SMTP server, listening on port 25.
+
+### Installation steps
+
+#### Create required users
+
+Create users `pmc` and `drop` with home directories:
+
+```shell
+sudo useradd -m -s /bin/bash pmc
+sudo useradd -m -s /bin/bash drop
+```
+
+If there is a list of users who should be able to log in as `drop` and/or `pmc` user through SSH,
+add /.ssh directories inside the newly created home directories of these users and put the list of SSH-RSA keys
+inside `authorized_keys` files (`home/pmc/.ssh/authorized_keys` and `home/drop/.ssh/authorized_keys`).
+
+#### Create required directories
+
+The following directories should be created for `pmc` user:
+- /home/pmc/data
+- /home/pmc/working_dir
+- /home/pmc/config
+- /home/pmc/libs
+
+```shell
+sudo -iu pmc
+cd /home/pmc
+mkdir data working_dir config libs
+```
+
+and for `drop` user:
+- /home/drop/drop_zone
+
+```shell
+sudo -iu drop
+cd /home/drop
+mkdir drop_zone
+```
+
+The `drop_zone` directory is where the pipeline checks for new data by default. It can be configured to be
+a symbolic link to the actual data directory, where the source data is delivered.
+
+To create a symlink, run:
+
+```shell
+ln -s /home/drop/drop_zone /home/drop/sample_test_data_folder/sample_dataset
+```
+
+#### Prepare the repositories
+
+Clone the indicated pipeline repository tag into the `pmc` user home directory.
+
+```shell
+sudo -iu pmc
+cd /home/pmc/
+git clone https://github.com/thehyve/pmc-conversion.git --branch <tag_name> --single-branch
+```
+
+Initialize git repositories inside `/home/pmc/data` and `/home/pmc/config` directories:
+```shell
+cd /home/pmc/data
+git init
+cd /home/pmc/config
+git init
+```
+
+#### Prepare external tools
+
+Download the latest version of the transmart-copy.jar from the Nexus repository of The Hyve:
+```shell
+curl -f -L https://repo.thehyve.nl/service/local/repositories/releases/content/org/transmartproject/transmart-copy/17.2.8/transmart-copy-17.2.8.jar -o transmart-copy.jar
+```
+
+Then put the transmart-copy.jar inside `/home/pmc/libs`.
+
+#### Prepare the configuration
+
+Prepare the [luigi.cfg](#luigi-configuration) and [email_config.cfg](#email-configuration)
+and put them into the `/home/pmc/pmc-conversion/` directory.
+
+
+#### Create a Python3 virtualenv
+
+```shell
+sudo -iu pmc
+cd /home/pmc
+python3 -m venv venv
+source venv/bin/activate
+python3 -m pip install -r /home/pmc/pmc-conversion/requirements/requirements.txt
```

Comment:
In the virtual environment the `pip` command is already set correctly, so you can use:
```suggestion
pip install -r /home/pmc/pmc-conversion/requirements/requirements.txt
```